src/cmd_line/commandLine.ts

```diff
@@ -17,6 +17,24 @@ class CommandLine {
    */
   private _commandLineHistoryIndex: number = 0;
 
+  /**
+   * for checking the last pressed key in command mode
+   *
```

Comment:
Extra star/carriage return on these comments.

Reply:
removed the extra star/carriage return